styles/base/inputs/_select.scss

```diff
@@ -20,6 +20,7 @@
 
   &:focus {
     border-color: $sprk-select-focus-border-color;
+    box-shadow: $sprk-select-focus-box-shadow;
```

Comment:
The selects in the deploy preview are missing the shadow: https://deploy-preview-3962--spark-sb-html.netlify.app/?path=/story/components-input-select--select-box